addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,273 @@
+        <label>Freebox Revolution Server</label>
+        <description>Provides various informations regarding the status of the Freebox Server</description>
+
+        <channels>
+            <channel id="fwversion" typeId="fwversion"/>
+            <channel id="uptime" typeId="uptime"/>
+            <channel id="restarted" typeId="restarted"/>
+            <channel id="tempcpum" typeId="tempcpum" />
+            <channel id="tempcpub" typeId="tempcpub" />
+            <channel id="tempswitch" typeId="tempswitch" />
+            <channel id="fanspeed" typeId="fanspeed" />
+            <channel id="reboot" typeId="reboot" />
+            <channel id="lcd_brightness" typeId="lcd_brightness" />
+            <channel id="lcd_orientation" typeId="lcd_orientation" />
+            <channel id="lcd_forced" typeId="lcd_forced" />
+            <channel id="wifi_status" typeId="wifi_status" />
+            <channel id="xdsl_status" typeId="xdsl_status" />
+            <channel id="line_status" typeId="line_status" />
+            <channel id="ipv4" typeId="ipv4" />
+            <channel id="rate_up" typeId="rate_up" />
+            <channel id="rate_down" typeId="rate_down" />
+            <channel id="bytes_up" typeId="bytes_up" />
+            <channel id="bytes_down" typeId="bytes_down" />
+            <channel id="onhook" typeId="onhook" />
+            <channel id="ringing" typeId="ringing" />
+            <channel id="call_number" typeId="call_number" />
+            <channel id="call_duration" typeId="call_duration" />
+            <channel id="call_timestamp" typeId="call_timestamp" />
+            <channel id="call_status" typeId="call_status" />
+            <channel id="call_name" typeId="call_name" />
+        </channels>
+         
+        <config-description>
+ 			<parameter name="ipAddress" type="text">
+ 				<label>Freebox Network Address</label>
+ 				<context>network_address</context>
+ 				<description>The IP address / FQDN of the Freebox Server (can include port number)</description>
+ 				<default>mafreebox.freebox.fr</default>
+ 				<required>false</required>
+ 			</parameter>
+ 
+			<parameter name="appToken" type="text">
+ 				<label>Application token</label>
+ 				<description>Token generated by the Freebox server</description>
+ 				<required>true</required>
+ 			</parameter>
+ 
+	 		<parameter name="refreshInterval" type="integer">
+ 				<label>Refresh Interval</label>
+ 				<description>The refresh interval in seconds which is used to poll given Freebox Server.</description>
+ 				<default>30</default>
+ 				<required>false</required>
+ 			</parameter>
+ 			
+ 	 		<parameter name="refreshPhoneInterval" type="integer">
+ 				<label>Phone Refresh Interval</label>
+ 				<description>The refresh interval in seconds which is used to poll given Freebox Server for phone informations.</description>
+ 				<default>2</default>
+ 				<required>false</required>
+ 			</parameter>
+ 
+ 		</config-description>
+    </thing-type>
+    
```

Comment:
You might want to flag many of the channels as "advanced", so that only the most important ones appear in the UI prominently. See https://github.com/eclipse/smarthome/blob/master/docs/sources/architecture/thing-definition.md#channels about the "advanced" flag.